src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+
+        if (userService.findByEmail(email) != null) {
+            User user = userService.findByEmail(email);
```

Comment:
Better call method `findByEmail` only once, not twice:
```suggestion
	User user = userService.findByEmail(email);
        if (user != null) {
```